src/components/MessageBar/MessageBar.scss

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE in the project root for license information.
+
+//
+// Office UI Fabric
+// --------------------------------------------------
+// MessageBar Styles
+
+/*
+  Base template file
+*/
+
+$MessageBar-padding: 8px;
+
+.ms-MessageBar {
+  padding: $MessageBar-padding;
+  display: table;
+
+  .ms-MessageBar-icon,
+  .ms-MessageBar-text {
+    display: table-cell;
+    vertical-align: top;
+  }
+
+  .ms-MessageBar-icon {
+    padding-right: $MessageBar-padding;
+    font-size: 16px;
+  }
+
+  .ms-MessageBar-text {
+    @include ms-font-s;
+  }
+
+}
+
+.ms-MessageBar--info {
+  @include ms-bgColor-info;
+
+  .ms-MessageBar-icon {
+    @include ms-fontColor-neutralSecondaryAlt;
+  }
+
+  .ms-Icon:before {
+    content: '\e00a';  // ms-Icon--infoCircle
```

Comment:
This should be moved into the HTML as `<i class="ms-Icon ms-Icon--infoCircle"></i>`